home.html

```diff
@@ -0,0 +1,327 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>TravelPro</title>
+    <link rel="stylesheet" href="style.css" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Nunito&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;300;400;500;600;700;800&display=swap"
+      rel="stylesheet"
+    />
+  </head>
+  <body>
+    <header>
+      <nav>
+        <section class="logo">
+          <img
+            src="elements\Decore.png"
+            alt="Pilania Travel Company Logo"
+            width="50"
+          />
+        </section>
+        <section class="nav-items">
+          <ul>
+            <li>Destinations</li>
+            <li>Hotels</li>
+            <li>Flights</li>
+            <li>Bookings</li>
+            <li>Login</li>
+            <li class="signup-btn">Sign up</li>
+            <li class="language-dropdown" style="font-size: smaller">
+              <span>EN&#9660;</span>
+              <ul class="dropdown-content">
+                <li>Hindi</li>
+                <li>German</li>
+              </ul>
+            </li>
+          </ul>
+        </section>
+      </nav>
+      <main>
+        <span class="left-section">
+          <h5>BEST DESTINATIONS AROUND THE WORLD</h5>
+          <h1>
+            Travel, enjoy <br />and live a new <br />
+            and full life
+          </h1>
+          <p>
+            Built Wicket Longer admire do barton vanity itself do in it. <br />
+            Preferred to sportsmen it engrossed listening. Park gate <br />
+            sell they west hard for the.
+          </p>
+          <button class="cta-button">Find out more</button>
+        </span>
+        <span class="right-section">
+          <img src="elements\traveller.png" alt="" />
+        </span>
+      </main>
+    </header>
+    <section class="category">
+      <div>
+        <header>CATEGORY</header>

```

Comment:
header tag should not use in section header has different meaning like footer

Reply:
resolved